### PR TITLE
additional details and regional for wafv2

### PR DESCRIPTION
**Pull request by Anarogk**
### Added
- New function `get_waf_v2_web_acl_details()` to get detailed information about individual WAF ACLs
- New function `get_waf_v2_web_acls_for_scope()` to handle get ACLs for different scopes (CLOUDFRONT/REGIONAL)
- Additional WAF ACL properties in Neo4j:
  - scope
  - default_action
  - rules_count
  - capacity

### CODE REVIEW

File: cartography/intel/aws/waf.py [hunk 1]
```diff
@@ -174,7 +238,11 @@ def _load_waf_v2_web_acls_tx(tx: neo4j.Transaction, web_acls: List[Dict], curren
     SET web_acl.lastupdated = $aws_update_tag,
         web_acl.name = record.Name,
         web_acl.region = record.region,
-        web_acl.consolelink = record.consolelink
+        web_acl.consolelink = record.consolelink,
+        web_acl.scope = record.scope,
+        web_acl.default_action = record.default_action,
+        web_acl.rules_count = record.rules_count,
+        web_acl.capacity = record.capacity,
```
Comment (ganeshjagdale9545) [inline cartography/intel/aws/waf.py:245]: Remove `,` from this line

**Review by ganeshjagdale9545 — CHANGES_REQUESTED** [review]
one minor change needed

File: cartography/intel/aws/waf.py [hunk 2]
```diff
@@ -119,44 +117,110 @@ def sync_waf_classic(
...
+) -> Dict:
+    """
+    Get detailed information about a WAFv2 Web ACL
+    """
+    try:
+        response = client.get_web_acl(Name=acl["Name"], Scope=scope, Id=acl["Id"])
+        acl_details = response.get("WebACL", {})
+
+        return {
+            "Name": str(acl["Name"]),
```
[the remark this answers is not in the document]
Comment (mpurusottamc) [reply cartography/intel/aws/waf.py:131]: @Anarogk can you resolve this?

[the remark this answers is not in the document]
Comment (Anarogk) [reply cartography/intel/aws/waf.py:131]: I was under the impression that all the properties we store in Neo4j should be strings.

**Review by mpurusottamc — APPROVED** [review]
lgtm

**Pull request merged**